### DISCUSSION TITLE
Maven Coordinates of Jipher

### DISCUSSION THREAD

**Discussion by mebigfatguy** [oracle/graal#7996] [General] [answered]
Can someone point me to the maven coordinates for the jipher jars?

**Comment by alina-yur** [comment, marked as answer]
hi @mebigfatguy, I don't think we release it on Maven, but I'll check. Would the Jipher JAR that is included in the GraalVM (Oracle GraalVM) build work for you? https://docs.oracle.com/en/graalvm/jdk/21/docs/security-guide/native-image/Jipher/#jipher-jce-with-native-image

**Comment by mebigfatguy** [comment]
Am curious, is there a reason not to? All the oci sdk jars are on maven. It just makes things infinitely easier.
